services/management/ia/templates_service.go

```diff
@@ -19,12 +19,12 @@ package ia
 import (
 	"bytes"
 	"context"
-	"html/template"
 	"io/ioutil"
 	"path/filepath"
 	"sort"
 	"strings"
 	"sync"
+	"text/template"
```

Comment:
Epic 😆 Please check other places too

Reply:
Already checked, seems that it the only place where we execute expression)